src/vivarium_gates_nutrition_optimization_child/components/population.py

```diff
@@ -105,12 +106,12 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
         self.register_simulants(new_simulants[self.key_columns])
 
         if pop_data.creation_time >= self.start_time:
-            ## Changing to run for a single subnational location
-            new_simulants["subnational"] = "Punjab"
-
-            # new_simulants["subnational"] = self._get_subnational_locations(
-            #     new_simulants.index
-            # )
+            if self.subnational == "All":
```

Comment:
Just so we are clear, the idea is you would replace all the subnationaals in the model spec and branches file with "all" instead of having to actually delete the key from the ocnfig files right?

Reply:
Yep! Exactly. If you put "All" it defaults to the old behavior and runs it "nationally" with the randomly assigned simulants.